dev-libs/benchmark/benchmark-1.3.0.recipe

```diff
@@ -1,14 +1,16 @@
 SUMMARY="A microbenchmark support library"
 DESCRIPTION="A library to support the benchmarking of functions, similar to unit-tests."
-HOMEPAGE="https://github.com/google/benchmark/"
+HOMEPAGE="https://github.com/google/benchmark"
 COPYRIGHT="2015-2017 Google Benchmark developers"
 LICENSE="Apache v2"
 REVISION="1"
 SOURCE_URI="https://github.com/google/benchmark/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="3dcc90c158838e2ac4a7ad06af9e28eb5877cf28252a81e55eb3c836757d3070"
+CHECKSUM_SHA256="f19559475a592cbd5ac48b61f6b9cedf87f0b6775d1443de54cfe8f53940b28d"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
 
-ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
-SECONDARY_ARCHITECTURES="?x86"
+libVersion="1.3.0"
```

Comment:
Thanks @DethroTull for the update. There are a few minor changes before we can merge this PR.

```diff
 CHECKSUM_SHA256="f19559475a592cbd5ac48b61f6b9cedf87f0b6775d1443de54cfe8f53940b28d"
+
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"

-libVersion="1.3.0"
+libVersion="0.0.0"
 libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
 
 PROVIDES="
 	benchmark$secondaryArchSuffix = $portVersion
-	lib:libbenchmark$secondaryArchSuffix = $portVersion
+	lib:libbenchmark$secondaryArchSuffix = $libVersionCompat
 	"
```

...

```diff
 PROVIDES_devel="
 	benchmark${secondaryArchSuffix}_devel = $portVersion
-	devel:libbenchmark$secondaryArchSuffix
+	devel:libbenchmark$secondaryArchSuffix = $libVersionCompat
 	"
```
(I'm not sure if this should be `$libVersionCompat` or just `$libVersion`. @alaviss, can you please confirm that it is appropriate to use `$libVersionCompat` here? Thanks :)

...
Later, after `BUILD_PREREQUIRES`, we could add a debuginfo package: 
```diff
 	cmd:make
 	"
+
+defineDebugInfoPackage benchmark$secondaryArchSuffix \
+	"$libDir"/libbenchmark.so.$libVersion
+
 PATCH()
 {
```
**but I'm not sure this would be appropriate.**
Anyway, if we choose to not add the debuginfo package then it would be better to compile with `-O2` and also strip the lib.
So I would go for a debuginfo package if nobody objects.